n_ua.c

```diff
@@ -25,7 +25,49 @@ static int n_cpu_mem = 0;
 static int n_cpu_mhz = 0;
 static int n_cpu_cores = 0;
 static char *n_cpu_vendor = NULL;
+
 static char *n_cpu_name = NULL;
+#if defined(ARDUINO_ARCH_ARC32)
```

Comment:
Why change the whitespace? Was this done by the linter?

Reply:
Yes. astyle removed the indentation.